cloudify_cli/cli/cfy.py

```diff
@@ -2450,6 +2434,35 @@ def blueprint_filter_rules(self, f):
     def deployment_filter_rules(self, f):
         return self._filter_rules(f, 'deployment')
 
+    @staticmethod
+    def secret_provider_type(required=True, _help=None):
+        args = [
+            '--type',
+            'secret_provider_type',
+        ]
+        kwargs = {
+            'required': required,
+            'help': _help or helptexts.SECRET_PROVIDER_TYPE,
+            'callback': validate_value_not_empty,
+        }
+
+        return click.option(*args, **kwargs)
+
+    @staticmethod
+    def connection_parameters(required=True, _help=None, default=None):
+        args = [
+            '--connection_parameters',
```

Comment:
eh maybe let's make this `--connection-parameters`? underscores in arguments are a bit weird i think